docker/Dockerfile

```diff
@@ -19,14 +19,22 @@ FROM ${BASE} AS scilla
 # run a copy -L to unfold the symlinkes, and strip all exes
 RUN mkdir -p /scilla/0/bin2/ && cp -L /scilla/0/bin/* /scilla/0/bin2/ && strip /scilla/0/bin2/*
 
-FROM rust:latest as rust_builder
-RUN rustup component add clippy rustfmt
-RUN apt-get update && apt-get install -y protobuf-compiler
+FROM ubuntu:18.04 as rust_builder
```

Comment:
What's the original problem and why we should build on ubuntu 18.04 base image instead of one coming from Rust?

Reply:
```
root@ip-10-3-37-186:/run/zilliqa# ldd /usr/local/bin/evm-ds                     
/usr/local/bin/evm-ds: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.28' not found (required by /usr/local/bin/evm-ds)
        linux-vdso.so.1 (0x00007ffd4815b000)
        libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f653cf55000)
        libpthread.so.0 => /lib/x86_64-linux-gnu/libpthread.so.0 (0x00007f653cd36000)
        libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f653c998000)
        libdl.so.2 => /lib/x86_64-linux-gnu/libdl.so.2 (0x00007f653c794000)
        libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f653c3a3000)
        /lib64/ld-linux-x86-64.so.2 (0x00007f653d562000)

Reply:
Got it, thanks!